test/lucas_kanade.jl

```diff
@@ -1,151 +1,125 @@
-# We output a message after loading each package to work around a
-# ten-minute timeout limit on Travis. Travis assumes the tests have hung
-# if the interval between printing something to stdio exceeds ten minutes.
 using Images
-@info "Finished loading Images package."
 using TestImages
-@info "Finished loading TestImages package."
 using StaticArrays
-@info "Finished loading StaticArrays package."
 using OffsetArrays
-@info "Finished loading OffsetArrays package."
 using Random
-@info "Finished loading Random package."
 using CoordinateTransformations
-@info "Finished loading CoordinateTransformations package."
 
-function evaluate_error(dims, flow::Array{SVector{2, Float64}, 1},  Δ,  tol)
+function evaluate_error(dims, flow::Array{SVector{2, Float64}, 1}, Δ, tol)
     error_count = 0
     maximum_error = 0.0
     for i in eachindex(flow)
         δ = flow[i]
         if abs(δ[1] - Δ[1]) > tol || abs(δ[2] - Δ[2]) > tol
-                error_count += 1
+            error_count += 1
         end
-        error = sqrt( (δ[1] - Δ[1])^2 + (δ[2] - Δ[2])^2 )
+        error = sqrt((δ[1] - Δ[1]) ^ 2 + (δ[2] - Δ[2]) ^ 2)
         if error > maximum_error
             maximum_error = error
         end
     end
     return error_count, maximum_error
 end
 
+function get_keypoints(image, n_points)
+    corners = imcorner(image, method=shi_tomasi)
+    I = findall(!iszero, corners)
```

Comment:
`findall` uses a for loop version, we can instead use broadcasting here, which is usually faster:

```suggestion
    I = LinearIndices(image)[corners .!≈ 0]
```

```julia
X = rand(64, 64);
J = randperm(prod(size(X)))[1:64];
X[J] .= 0;

I1 = findall(!iszero, X);
I2 = CartesianIndices(X)[X .!= 0];
I1 == I2 # true

@btime findall(!iszero, $X); # 34.101 μs (13 allocations: 128.61 KiB)
@btime CartesianIndices($X)[$X .!= 0] # 17.237 μs (5 allocations: 67.92 KiB)
```

---

Edit: oops this is test codes. Well, feel free to ignore this suggestion. But do keep in mind the differences.